model/project_parser.go

```diff
@@ -464,6 +486,13 @@ func (pss *parserStringSlice) UnmarshalYAML(unmarshal func(interface{}) error) e
 	return nil
 }
 
+func mergeParserStringSlice(main, add parserStringSlice) parserStringSlice {
```

Comment:
i'm not receiving an error for `pp1.Ignore = append(pp1.Ignore, pp2.Ignore...)`; are you sure this is necessary?